classes/order/OrderInvoice.php

```diff
@@ -359,6 +359,9 @@ public function getProductTaxesBreakdown($order = null)
         if ($sum_composite_taxes) {
             $grouped_details = [];
             foreach ($details as $row) {
+                if ($this->id !== (int) $row['id_order_invoice']) {
+                    continue;
+                }
```

Comment:
I wonder if this is a breaking change or not 🤔 

With this PR I'm changing the behavior of the method (which was wrong), so if let's say a module was using this method expecting the previous wrong behavior, it could be perceived as a breaking change.

Reply:
I think the fix is correct, this method didn't take into account the possibility of multi invoices per Order So this fixes the (incorrect) behaviour for multi invoices, but it doesn't change it for single invoice case
Hence it shouldn't break any behaviour